Core/Inc/control.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file control.h
+ * @author Scott Abramson
+ * @brief Tasks for controlling the car.
+ * @version 0.1
+ * @date 2024-08-04
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#ifndef CONTROL_H
+#define CONTROL_H
+
+#define BRAKE_STATE_UPDATE_FLAG 0x00000001U
+
+/**
+ * @brief Task for controlling the brakelight.
+ */
+void vBrakelightControl(void *pv_params);
```

Comment:
I made it a thread since it gets its data from pedal data processing and I didn't want to include it in pedal processing since I want threads to be divided by physical part of car.